tests/OSPSuite.Core.Tests/Services/PopulationDataSplitterSpecs.cs

```diff
@@ -102,37 +101,37 @@
 //
 //   internal class When_filling_parameters_and_initial_values_for_the_first_individual : concern_for_PopulationDataSplitterSpecs
 //   {
-//      private IList<IParameterProperties> _variableParameters;
-//      private IList<ISpeciesProperties> _variableInitialValues;
+//      private IList<ParameterProperties> _variableParameters;
+//      //TODO  private IList<ISpeciesProperties> _variableInitialValues;
 //
 //      protected override void Context()
 //      {
 //         base.Context();
 //
-//         _variableParameters = new List<IParameterProperties>();
+//         _variableParameters = new List<ParameterProperties>();
 //
-//         var parameterInfo1 = A.Fake<IParameterProperties>();
-//         A.CallTo(()=>parameterInfo1.IsTable).Returns(false);
+//         var parameterInfo1 = A.Fake<ParameterProperties>();
```

Comment:
Because we have no interface, I won't be able to fake those guys, 
I will rewrite thsoe tests completely to be interactiontests. But first, the PopRunner needs to work